layer/index.js

```diff
@@ -57,8 +58,8 @@ router.route('/:layerId')
             label ??= label ?? layer.label
             if(canvases?.length === 0) canvases = undefined
             const updatedLayer = canvases ?
-                Layer.build(projectId, label, canvases)
-                : new Layer(projectId, {id:layer.id, label, pages:layer.pages})
+                Layer.build(projectId, label, canvases, req.user._id)
```

Comment:
`creator` isn't the `req.user._id` it is the agent from that token. It may be handy in the object, but can also be assembled without trouble as we worked hard to make the hexcodes match.

Reply:
It is only when the creator goes to RERUM that it must be a URI, so we can change it at either end.